src/main/resources/template/cat-base-detail.tpl.html

```diff
@@ -6,7 +6,7 @@
     <div class="panel-heading">
         <div class="pull-right edit-buttons" ng-if="!editDetail && !!editTemplate">
             <div class="btn-group">
-                <button type="button" class="btn btn-xs btn-default" cat-confirm-click cat-on-confirm="remove()"
+                <button type="button" class="btn btn-xs btn-default" cat-confirm-click="{{catConfirmDeleteMessage}}" cat-on-confirm="remove()"
```

Comment:
where does catConfirmDeleteMessage come from?
as far as i can see it will always be empty by default, and doe to the fact that we only access the attributes, the confirm message now will defualt to `{{catConfirmDeleteMessage}}` (unevaluated) and the real fallback `'Are you sure?'` will never be used